docs/tempo/website/configuration/_index.md

```diff
@@ -720,29 +713,28 @@ overrides:
     # Global ingestion limits configurations
 
     # Burst size (bytes) used in ingestion.
-    # (Default: `20,000,000` ~20MB )
     # Results in errors like
     #   RATE_LIMITED: ingestion rate limit (15000000 bytes) exceeded while adding 10 bytes
-    [ingestion_burst_size_bytes: <int>]
+    [ingestion_burst_size_bytes: <int> | default = 20_000_000 (20MB) ]
```

Comment:
Integers with underscores are perfectly ok to be parsed by golang. Underscores were intentionally introduced to make reading easier. You can find several examples of its use in our files. An example is: https://github.com/grafana/tempo/blob/ba93a400a4968c303cdbff9f5ccf88f6194489ca/example/docker-compose/s3/tempo-s3.yaml#L27

However, I will proceed to accept your change on this PR since there is a human readable value in brackets.